awl/mslider.cpp

```diff
@@ -229,20 +229,21 @@ void MeterSlider::paintEvent(QPaintEvent* ev)
       //    draw tick marks
       //---------------------------------------------------
 
-  	QFont f(p.font());
-   	f.setPointSize(6);
-   	p.setFont(f);
+  	  QFont f(p.font());
+   	  f.setPointSize(6);
+   	  p.setFont(f);
       p.setPen(QPen(Qt::darkGray, 2));
-   	QFontMetrics fm(f);
+   	  QFontMetrics fm(f);
       int xt = 20 - fm.width("00") - 5;
+      y1++;
 
       QString s;
-   	for (int i = 10; i < 70; i += 10) {
-      	h  = y1 + lrint(i * mh / range);
-         	s.setNum(i - 10);
-  		p.drawText(xt,  h - 3, s);
-		p.drawLine(15, h, 20, h);
-         	}
+   	  for (int i = minValue(); i < (maxValue() - minValue()); i += 10) {
+      		h  = y1 + lrint(i * mh / range);
+        	s.setNum(-1 * (i - 20));
```

Comment:
I would change the above 3 lines into:
```
       for (int i = minValue() + 10; i < maxValue(); i += 10) {
            h = y1 + lrint((maxValue() - i) * mh / range);
            s.setNum(i);
```
The way you had it, it was actually iterating all the way to maxValue() - minValue() which evaluates to (20 - (-60) which equals 80.  So iteration range was from -20 to 80, which is wider than the range of the slider.  That setNum line was trying to adjust for the offset.  Anyway, I think better to do it as I've shown.  It will also cause the highest notch to end before the "20", which was getting cut off.  So this is how my code change displays:

![screenshot 190](https://cloud.githubusercontent.com/assets/6502474/24066750/9f2e8720-0b4b-11e7-908f-f1b459347946.png)